src/ctia/http/routes/crud.clj

```diff
@@ -331,14 +333,18 @@
              :description (capabilities->description search-capabilities)
              :capabilities search-capabilities
              :query [params search-q-params*]
-             (-> (get-store entity)
-                 (store/query-string-search
-                   (search-query date-field params)
-                   identity-map
-                   (select-keys params routes.common/search-options))
-                 (ent/page-with-long-id services)
-                 ent/un-store-page
-                 routes.common/paginated-ok))
+             (let [params* (if (= "true" (flag-value :enforce-search-fields))
+                             (routes.common/enforce-search-fields
+                              params searchable-fields)
+                             params)]
```

Comment:
This part here is only temporarily. Semantically, enforcing fields should be happening at the lower level, somewhere within `ctia.stores.es.crud/make-search-query`, making it happen down at that level requires changing the signature of every instance of `IQueryStringSearchableStore/query-string-search`, so parts of `entity-crud-config` could hike down there as well (in this case `searchable-fields` and `FeaturesService` instance). 

This may initially create more mess than just sticking it in here. But eventually that's what needs to be done.